### ISSUE TITLE
BUG: Cannot export Points in COCO Format

### ISSUE THREAD

**Issue by JParzival** [openvinotoolkit/cvat#2410]
Hello everyone,

the issue itself is that, when I export annotations in COCO Format, points are not shown in the json, but polygons are.
I have tried both creating a new task and only annotating points and also combining points and polygons, and neither of them exports the COCO Format with both instances.

### Expected Behaviour
The expected behaviour is to be able to export the COCO Format with both points and polygons.

### Current Behaviour
The current behaviour is that the COCO Format is not able to export points.

### Possible Solution
It would just be a matter of implementing the points inside the COCO Format, so that they can be exported alongside the polygons.

### Steps to Reproduce (for bugs)
1. Create a new task
2. Set a point
3. Export in COCO Format

### Context
The project that I am actually working in requires both a combination of a mask and a keypoint inside of it, in the most important part of the object. Without the points, the goal cannot be accomplished.

### Your Environment

- Docker version `docker version` (e.g. Docker 17.0.05): Docker version 19.03.12, build 48a66213fe
- Are you using Docker Swarm or Kubernetes? No, just Docker (required for the application)
- Operating System and version (e.g. Linux, Windows, MacOS): Windows
- Code example or link to GitHub repo or gist to reproduce problem: https://github.com/openvinotoolkit/cvat

**Comment by claverru** [comment]
I'm also interested on this.

**Comment by taranais** [comment]
I am looking for this

**Comment by zhiltsov-max** [comment]
COCO keypoints are not supported yet, possible solutions are described here: https://github.com/openvinotoolkit/cvat/issues/1784

**Comment by claverru** [comment]
> COCO keypoints are not supported yet, possible solutions are described here: #1784 and [#1421 (comment)](https://github.com/openvinotoolkit/cvat/issues/1421#issuecomment-616576420)

@zhiltsov-max I think those workarounds don't work for me because I have less than 3 keypoints, and masks are only supported with at least 3 points.

**Comment by claverru** [comment]
@zhiltsov-max Also, I think having keypoints in COCO format should be a very common feature.

**Comment by JParzival** [comment]
Yes, I agree with @claverru, having the keypoints in COCO is a very common feature, and in my opinion is an urgent enhancement

**Comment by zhiltsov-max** [comment]
https://github.com/openvinotoolkit/cvat/issues/1421#issuecomment-616576420 describes exactly this topic.
